lib/engine/game/g_1835/round/operating.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative '../../../round/operating'
+
+module Engine
+  module Game
+    module G1835
```

Comment:
Don't override if unnecessary to do so

Reply:
without this file, there is not operating round possible, only stockrounds.

Reply:
you can just use the generic one, you don't need to subclass if it's unaltered.

Reply:
i found the right place.